processes/rescale_chunks.json

```
@@ -2,7 +2,9 @@
     "id": "rescale_chunks",
     "Summary": "Rescaling of RGB within chunk of polygons.",
     "description": "A simple process for rescaling Sentinel 2 RGB images within polygon chunks that also showcases how to use ``chunk_polygon() with a (User Defined Function) UDF.",
-    "Categories": ["arrays"],
+    "Categories": [
```

```suggestion
    "categories": [
```

Generally lower-case.

---

i think i tried with small initially, but I can give a try again. Similar was with Summary

---

Sorry, it does work for categories. But not for summary

---

For summaries it may complain about the dots at the end or the length. 

Otherwise, I can give it a try later.

---

Indeed, it was either casing issues or the dots at the end of the summaries.
The linter error messages were not very helpful, unfortunately. Something to improve in the linter code... :-)